src/arrayMethodIndexOf.test.js

```diff
@@ -48,3 +48,8 @@ test(`[].indexOf2() = -1`, () => {
   expect([].indexOf2())
     .toBe(-1);
 });
+
+test(`indexOf2(44, 4) = 4 fromIndex 4`, () => {
```

Comment:
description should be human readable. 
"returns element when it's on a fromIndex position"

let's update for other test case description to be human readable too.

Reply:
done